d2layouts/d2near/layout.go

```diff
@@ -134,6 +229,9 @@ func boundingBox(g *d2graph.Graph) (tl, br *geo.Point) {
 				y2 = math.Max(y2, obj.TopLeft.Y+obj.Height)
 			}
 		} else {
+			if obj.OuterNearContainer() != nil {
```

Comment:
what's this for?

Reply:
once we append a near container inside the main layout, those objects inside a near container(which doesn't have near attribite either) shouldn't affect the way we calc bounding box

Reply:
oh i see. in the same way that any object inside any container won't affect bounding box? or something different?

Reply:
it depends on our requirement, for this case, i think keep bounding box the original one should be correct way. consider this case below:
```
x -> y

a: {
  near: top-center
  c
}
b: {
  near: top-right
  c
}
```

if we don't skip the `c` object inside `a`, after append `a` and `c` inside the main diagram, when it's turn to place `b` object, the new bounding box (blue colore) will be larger than the original one(red color), and so `top-right` will be placed higher than `top-center`
<img width="994" alt="image" src="https://user-images.githubusercontent.com/19243114/230526852-e48767c3-e154-448f-be55-7da97ce5fcda.png">

keep the bounding box the original one when place all the near objects will make more sense here
<img width="539" alt="image" src="https://user-images.githubusercontent.com/19243114/230527073-0c5e2f63-6259-42e6-8b48-444ba04df2b8.png">